src/proto/FunctionRpc.proto

```diff
@@ -298,6 +304,12 @@ message RpcFunctionMetadata {
 
   // A flag indicating if managed dependency is enabled or not
   bool managed_dependency_enabled = 14;
+
+  // Indexing mode for the function 
+  FunctionIndexingMode = 15;
```

Comment:
Unless I am missing something, we will need to declare variable as - `FunctionIndexingMode function_indexing_mode = 15;`

Reply:
Below field also does the same thing if we should think about leveraging it - 
https://github.com/Azure/azure-functions-language-worker-protobuf/blob/6635670b136d565f2a196ce1a64fee1ea5c9c1c9/src/proto/FunctionRpc.proto#L317-L318

Reply:
Good point. We can just use the use_default_metadata_indexing property. I have removed the other property which I added.